src/Header.jsx

```diff
@@ -0,0 +1,3 @@
+export const Header = (props) => {
```

Comment:
Destructuring is a good approach. It will be good you get used to it from now on.

```suggestion
export const Header = ({course}) => {
```